tests/unit/test_eq.py

```diff
@@ -56,6 +57,13 @@ def test_create_eq_constructor_missing_uprn(self):
             self.assertIn(f'Could not retrieve address uprn from case JSON',
                           ex.exception.message)
 
+    @staticmethod
+    async def patch_response_id(payload, eq_payload):
```

Comment:
this is the wrong way to do it .  We are just hacking results to be the same, so it is no kind of test.